MOBILE_CLIENT/ANDROID/_IMPORTANT/APK_SCREENSHOT_DISABLED/meta.json

```diff
@@ -0,0 +1,12 @@
+{
+  "risk_rating": "important",
+  "short_description": "Application disables taking screenshots, preventing call coverage from computing coverage.",
```

Comment:
This is redudent: `preventing call coverage from computing coverage.`

Reply:
Reworded.